### DISCUSSION TITLE
Cannot open 'MagickCore/magick-baseconfig.h' No such file or directory.

### DISCUSSION THREAD

**Discussion by YusufSuleman** [ImageMagick/ImageMagick#4063] [Help]
I can't find this file anywhere in the ImagMagick zip.

Screenshots:
![image](https://user-images.githubusercontent.com/63107868/128817995-7056a3b5-6047-45d2-8e5f-dca497a860db.png)
Only one because there are hundreds of errors.

**Comment by snibgo** [comment]
That file is created by "configure", a bash script. It contains information that is specific to your build of IM, eg whether various delegates have been linked in.

**Reply by YusufSuleman** [reply]
@snibgo But I don't use Linux, How do I run the bash script.

**Comment by snibgo** [comment]
How did you build IM? I expect magick-baseconfig.h should have been built as part of that process.
